dao-contracts/src/voting/mod.rs

```diff
@@ -0,0 +1,9 @@
+pub mod governance_voting;
+pub mod vote;
```

Comment:
There is no need to make the whole module public. Below you reimport all structs/functions you want to expose
```suggestion
mod governance_voting;
mod vote;
```